packages/terra-dropdown-button/src/DropdownButton.jsx

```diff
@@ -169,6 +176,13 @@ class DropdownButton extends React.Component {
       theme.className,
     );
 
+    let buttonAriaLabel = '';
+    const modifiedButtonAttrs = { ...buttonAttrs };
+    if (modifiedButtonAttrs && modifiedButtonAttrs['aria-label']) {
+      buttonAriaLabel = modifiedButtonAttrs['aria-label'];
+      delete modifiedButtonAttrs['aria-label'];
+    }
+
```

Comment:
Nit: 
`buttonAriaLabel` can be constructed here instead of line no. 211. for better readablity
```js
const customLabel =  (selectText) ? `${selectText}, ${selectedLabel}, ${label}` :  label;
buttonAriaLabel = customLabel + buttonAriaLabel;
```
since buttonAttrs is initialised with '' no need for check before using it.